### DISCUSSION TITLE
Add tGeneric6DofSpring2Constraint support for btMultiBody

### DISCUSSION THREAD

**Discussion by erwincoumans** [bulletphysics/bullet3#3498] [General]
Right now, btMultiBody has very limited support for constraints, mainly contact constraints,btMultiBodyPoint2Point and btMultiBodyJointMotor.

Implement support for btGeneric6DofSpring2Constraint, then we cover most cases.

**Comment by benelot** [comment]
This might be relevant in the same run:
#463 Implement btMultiBodyJointMotor and btMultiBodyJointLimitConstraint for Spherical Joint
